From da25423d253fa77b91db23cfb8094de748b46703 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 19 Apr 2021 20:11:37 +0200
Subject: [PATCH] host-pip-requirements: add reForis distutils

According to:
https://github.com/openwrt/packages/tree/master/lang/python#installing-host-side-python-packages

This allows us to specify hash for reForis distutils on PyPI
and once we specified hash, we can then set Host package in Makefile of
reForis

This patch can not be accepted in upstream.
See: https://github.com/openwrt/packages/pull/14329

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/python/host-pip-requirements/reforis-distutils.txt | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 lang/python/host-pip-requirements/reforis-distutils.txt

diff --git a/lang/python/host-pip-requirements/reforis-distutils.txt b/lang/python/host-pip-requirements/reforis-distutils.txt
new file mode 100644
index 000000000..9e8e3b0f1
--- /dev/null
+++ b/lang/python/host-pip-requirements/reforis-distutils.txt
@@ -0,0 +1,3 @@
+reforis-distutils==0.1.0 --hash=sha256:f95432eca1bf4faced68d20d93e00fbdd0ae63801db4c701a42d0738cd0db37b
+Babel==2.12.1 --hash=sha256:cc2d99999cd01d44420ae725a21c9e3711b3aadc7976d6147f622d8581963455
+pytz==2023.3 --hash=sha256:1d8ce29db189191fb55338ee6d0387d82ab59f3d00eac103412d64e0ebd0c588
-- 
2.25.1

